### DISCUSSION TITLE
Private Lists: The Most Wanted Feature. 🗒️

### DISCUSSION THREAD

**Discussion by michael-azogu** [community/community#42377] [Other Feature Feedback, Questions, & Ideas] [answered]
Product Feedback

I'm a lists power user & I've been waiting for private lists all year.

Please just prioritize it.
![Screenshot_20221222-134713_Opera](https://user-images.githubusercontent.com/73001497/209138867-72fab462-347d-47ac-9dca-5ed69fccaa5c.png)

**Comment by michael-azogu** [comment]


**Comment by jfullstackdev** [comment]
what would be the benefit of having Private Lists when you can simply add private repos in it and it will not be seen by others

**Reply by michael-azogu** [reply]
I mean for lists of other repos I don't own

**Comment by thillainirmal-tech** [comment]
Please make elements should <space string nothing> is else floating point with #integrity  "require-directory": "^2.1.1",

**Comment by alexanaro1217** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I'm a lists power user & I've been waiting for private lists all year.
> 
> Please just prioritize it.
> ![Screenshot_20221222-134713_Opera](https://user-images.githubusercontent.com/73001497/209138867-72fab462-347d-47ac-9dca-5ed69fccaa5c.png)
>

**Reply by Santiagosaiz** [reply]
31368e401b4366b91395c1d9bac1ff7a9dab30df

**Comment by Slava848** [comment]
f

**Comment by Santiagosaiz** [comment]
31368e401b4366b91395c1d9bac1ff7a9dab30df

**Comment by AIRCOURIELtd** [comment]
> 
> 
> Product Feedback
> 
> I'm a lists power user & I've been waiting for private lists all year.
> 
> Please just prioritize it.
> ![Screenshot_20221222-134713_Opera](https://aircourieinternationalservice.com-images.githubusercontent.com/73001497/209138867-72fab462-347d-47ac-9dca-5ed69fccaa5c.png)
>

**Comment by AhmTaher** [comment]
No problem at all! I'm here to help. Please go ahead and provide the additional details or context, and I'll do my best to assist you with an answer. 😊

**Comment by Syedmdreyazul** [comment]
Yes.

**Comment by LiteBrite82** [comment, marked as answer]
Hi @michael-azogu 

This has been requested in other discussions as well:
[Make lists either private, registered and public](https://github.com/orgs/community/discussions/8766)

I'm going to close this discussion as it is a duplicate; your feedback has been counted!
